src/blocks/RenderBlocks.tsx

```diff
@@ -20,6 +21,7 @@ const blockComponents = {
   mediaBlock: MediaBlock,
   productsList: ProductsListBlock,
   faq: FAQBlock,
+  tabCards: TabCards
```

Comment:
```suggestion
  tabCards: TabCards,
```
The missing comma currently breaks the build.